src/usr/local/www/widgets/widgets/system_information.widget.php

```diff
@@ -221,8 +218,7 @@ function get_item_html($itemkey) {
 	</div>
 	<span id="mbufusagemeter">%s</span>&nbsp;<span id="mbuf">(%s)</span>
 END;
-			$mbufstext = get_mbuf();
-			$mbufusage = get_mbuf(true);
+			get_mbuf($mbufstext, $mbufusage);
```

Comment:
get_mbuf() has a newer interface to make a single call and get both items of data.